src/sentry/features/temporary.py

```diff
@@ -29,14 +29,21 @@ def register_temporary_features(manager: FeatureManager):
 
     # NOTE: Please maintain alphabetical order when adding new feature flags
 
-    # Features that don't use resource scoping
-    manager.add("auth:register", SystemFeature, FeatureHandlerStrategy.INTERNAL)
+    # Features that don't use resource scoping #
+    ############################################
+
+    # Enables user registration.
+    manager.add("auth:register", SystemFeature, FeatureHandlerStrategy.INTERNAL, default=True)
     manager.add("organizations:create", SystemFeature, FeatureHandlerStrategy.INTERNAL)
     manager.add("organizations:multi-region-selector", SystemFeature, FeatureHandlerStrategy.INTERNAL)
     manager.add("relocation:enabled", SystemFeature, FeatureHandlerStrategy.INTERNAL)
 
-    # Organization scoped features that are in development or in customer trials.
+    # Organization scoped features that are in development or in customer trials. #
+    ###############################################################################
+
+    # Enables activated alert rules
```

Comment:
Were you thinking of porting over all the comments.

At this point maybe features themselves should just have a `description` hah

Reply:
Oops sorry missed your comment here https://github.com/getsentry/sentry/pull/71828#discussion_r1622684629

Reply:
Making the comment into a feature description isn't a bad idea 🤔